src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    static final String WORD_START_W = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder lines = new StringBuilder();
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            String value = reader.readLine();
+            while (value != null) {
+                lines.append(value.toLowerCase()).append(System.lineSeparator()).append(" ");
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException();
```

Comment:
```suggestion
            throw new RuntimeException(e);
```
let's make the RuntimeException informative